examples/seismic/model.py

```diff
@@ -201,6 +244,30 @@ def demo_model(preset, **kwargs):
         return Model(space_order=space_order, vp=v, origin=origin, shape=v.shape,
                      dtype=np.float32, spacing=spacing, nbpml=20)
 
+    elif preset.lower() in ['marmousi-elastic', 'marmousi2d-elastic']:
+        shape = (1601, 401)
+        spacing = (7.5, 7.5)
+        origin = (0., 0.)
+
+        # Read 2D Marmousi model from opesc/data repo
+        data_path = kwargs.get('data_path', None)
+        if data_path is None:
+            error("Path to opesci/data not found! Please specify with "
+                  "'data_path=<path/to/opesci/data>'")
```

Comment:
I'd again drop ``error`` and just raise the ValueError exception (perhaps suggesting to set ``data_path`` in there)

Reply:
btw, ``data_path`` is quite important, so perhaps the docstring of ``demo_model`` should be refreshed at some point

Reply:
Same as it was before for acoustic-marmousi

Reply:
like here (raise+error)